subgraphs/compound-forks/protocols/bastion-protocol/src/mapping.ts

```diff
@@ -125,6 +153,9 @@ export function handleMarketListed(event: MarketListed): void {
       cTokenReserveFactorMantissa
     );
     _handleMarketListed(marketListedData, event);
+    const market = Market.load(cTokenAddr.toHexString())!;
```

Comment:
It is possible there is an early return in _handleMarketListed() so we should error check this val

Reply:
I would say we should crash if the market is not successfully created in the line above (early return); if not here, certainly later somewhere else.

Reply:
At both instances in this function can we do an error handling like this: https://github.com/messari/subgraphs/blob/5b3b0c500856a8e79c481ddfb3068ce4413efc80/subgraphs/compound-forks/protocols/bastion-protocol/src/mapping.ts#L382

we can do a `log.critical` error in order to crash the subgraph

For more context, we are in the process of planning to improve our data quality. One of the pieces of this is not silently failing and not crashing to create downtime. We will begin to raise errors as they happen without silently failing like rn.

Either way, I am going to be strict on proper error handling. `log.critical` is just easier to find in code than a quiet `!`